main.go

```diff
@@ -216,6 +216,22 @@ func exportArtifacts(srcDir, destDir string, copiedHandler func(src, dest string
 	return nil
 }
 
+func logExitStatus(exitStatus int) {
+	statusCodes := map[int]string{
```

Comment:
The list comes from the Flank GitHub page: https://github.com/Flank/flank

Reply:
NOTE: might worth adding to the code as a comment, so that we can find the source later: https://github.com/Flank/flank#exit-codes